aat/build.gradle

```
@@ -24,6 +24,8 @@ dependencies {
     testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.24.2'
     testImplementation group: 'com.github.hmcts', name: 'befta-fw', version: '9.2.0'
     testImplementation 'com.github.hmcts:fortify-client:1.4.2'
+    testImplementation 'com.github.hmcts:fortify-client:1.4.8:all'
```

can we remove testImplementation 'com.github.hmcts:fortify-client:1.4.2' so it is only defined once 
```suggestion
    testImplementation 'com.github.hmcts:fortify-client:1.4.8:all'
```